crates/holochain/src/conductor/api/api_cell.rs

```diff
@@ -125,4 +135,28 @@ pub trait CellConductorApiT: Clone + Send + Sync + Sized {
 
     /// Get a [EntryDef] from the [EntryDefBuf]
     async fn get_entry_def(&self, key: &EntryDefBufferKey) -> Option<EntryDef>;
+
+    /// Turn this into a call zome handle
+    fn into_call_zome_handle(self) -> CallZomeHandle;
+}
+
+#[async_trait]
+/// A handle that cn only call zome functions to avoid
+/// making write lock calls
+pub trait CallZomeHandleT: Send + Sync {
+    /// Invoke a zome function on a Cell
+    async fn call_zome(
+        &self,
+        invocation: ZomeCallInvocation,
+    ) -> ConductorApiResult<ZomeCallInvocationResult>;
+}
```

Comment:
If we extrapolated this, we'd need a different handle type for every method on the conductor. Perhaps there can be a ConductorHandleRead which only has the read stuff, similar to EnvironmentRead/Write?

Reply:
Yeh I could get round that. I think this is still going to need different functions though because we are going to have to somehow pass in the workspace if it's the same dna.